test/runtime/getUrl.test.js

```diff
@@ -127,5 +127,6 @@ describe("escape", () => {
         { hash: "#hash", needQuotes: true },
       ),
     ).toMatchSnapshot();
+    expect(getUrl("http://url/path\\/")).toMatchSnapshot();
```

Comment:
Can you provide a real example of the problem? Using `url()`?

Reply:
@alexander-akait I've added a more real URL example. Hope that's good enough. Thanks!